src/web/src/pages/CourseExplorer.vue

```diff
@@ -56,9 +56,13 @@
       </b-col>
     </b-row>
   </b-container>
-  <div v-else>
-    <b-spinner></b-spinner>
-    <strong class="m-2">Loading courses...</strong>
+  <div id="spinnerWrapper" v-else>
```

Comment:
Do you need this div here? Seems like you might be able to just do `<CenterSpinner v-else`. If the div is needed though you should still remove the `spinnerWrapper` ID as its not being used. 

This comment also applies to the CoursePage, SubjectExplorer, and CourseScheduler

Reply:
I forgot that vue3 doesn't require you to have only one parental directory under template, updated.